src/components/Tabs/Tabs.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Base from '../Base.jsx';
+import style from './style.scss';
+import cn from 'classnames';
+import _ from 'lodash';
+
+class Tabs extends React.Component {
+  constructor(props) {
+    super(props);
+
+    let selectedIndex = _.findIndex(props.items, item => item.selected);
```

Comment:
minor: maybe the lodash shorthand exists for this: `_.findIndex(props.items, 'selected')`